pkgs/development/python-modules/vdf/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
+, mock }:
+
+buildPythonPackage rec {
+  pname = "vdf";
+  version = "3.1";
+
+  src = fetchFromGitHub {
+    owner = "ValvePython";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "19xqjq2159w2l9vaxlkickvy3zksp9ssdkvbfcfggxz31miwp1zr";
+  };
+
+  checkInputs = [ mock ];
```

Comment:
This probably needed, pytest and pytest-cov no?

Reply:
It seemed to run the tests fine without them. I will look into it further.

Reply:
see

>You can do so by overriding the checkPhase to run
> https://github.com/ValvePython/vdf/blob/master/Makefile#L8

Python packages don't run `make`.